File: python/P2469_ConvertTheTemperature.py
from typing import List

class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
        kelkin = celsius + 273.15
        fahrenheit = celsius * 1.80 + 32
        return [kelkin, fahrenheit]

    
solution = Solution()
print(solution.convertTemperature(36.50))

File: kotlin/P2469_ConvertTheTemperature.kt
fun convertTemperature(celsius: Double): DoubleArray {
    val kelvin = celsius + 273.15
    val fahrenheit = celsius * 1.80 + 32.00
    
    return doubleArrayOf(kelvin, fahrenheit)
}

fun main() {
    val celsius1 = 36.50
    val celsius2 = 122.11

    println("Input: celsius = $celsius1")
    println("Output: ${convertTemperature(celsius1).joinToString(", ")}")
    
    println("Input: celsius = $celsius2")
    println("Output: ${convertTemperature(celsius2).joinToString(", ")}")
}